DotNet/DataAcquisition.AcquisitionWorker/appsettings.json

```diff
@@ -0,0 +1,67 @@
+{
+  "DistributedLockSettings": {
+    "Expiration": 10,
+    "RetryDelay": 5,
+    "MaxRetryCount": 3,
+  },
+  "ServiceInformation": {
+    "Name": "Link Data Acquisition Worker Service",
+    "ServiceName": "DataAcquisition.AcquisitionWorker",
+    "Version": "0.1.1-dev",
+    "GitCommit": "git_value"
+  },
+  "EnableSwagger": false,
+  "KafkaConnection": {
+    "BootstrapServers": [ "" ],
+    "SaslProtocolEnabled": true,
+    "SaslUsername": ""
+  },
+  "DatabaseProvider": "SqlServer",
+  "ConnectionStrings": {
+    "DatabaseConnection": "",
+    "RedisConnection": "",
+  },
+  "Telemetry": {
+    "EnableTelemetry": false,
+    "EnableRuntimeInstrumentation": false,
+    "EnableOtelCollector": false,
+    "OtelCollectorEndpoint": "",
+    "EnableAzureMonitor": false
+  },
+  "ConsumerSettings": {
+    "ConsumerRetryDuration": [ "PT20S", "PT60S", "PT120S" ],
+    "DisableRetryConsumer": false,
+    "DisableConsumer": false
+  },
+  "Serilog": {
+    "Using": [ "Serilog.Sinks.Console", "Serilog.Sinks.Grafana.Loki" ],
+    "MinimumLevel": {
+      "Default": "Information"
+    },
+    "WriteTo": [
+      {
+        "Name": "GrafanaLoki",
+        "Args": {
+          "uri": "",
+          "labels": [
+            {
+              "key": "app",
+              "value": "link-cloud"
+            },
+            {
+              "key": "component",
+              "value": "DataAcquisition"
```

Comment:
Do you think it would be worth changing this to match the service name you gave it? DataAcquisition.AcquisitionWorker or just AcquisitionWorker?